languages/swift/exercises/concept/functions/.docs/hints.md

```diff
@@ -0,0 +1,34 @@
+## 1. Default parameter values
+
+- [Default parameter values][default-parameter-values] can be specified by adding `= value` after the parameter's type annotation. Where _value_ is the default you want to use
```

Comment:
This hint gives away the solution in its text. Could you perhaps rephrase to not directly give away the solution?

This comment also applies to some of the other hints.